gitpod-io/gitpod
# [jb] respect GW user settings (proxies, certificates)

## Description


Respect user settings (proxies, certificates) of JetBrains Gateway while connecting to Gitpod API.

## Related Issue(s)

fix #10694

## How to test


Test for regressions:
- download a version of JB Gateway plugin from JB Marketplace [[1](https://plugins.jetbrains.com/plugin/download?rel=true&updateId=189730)]
- install  it manually following this instruction [[2](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk)]
  - make sure to click apply and restart IDE
- check whether that you still can see list of workspaces on gitpod.io
- start a workspace in preview envs with IntelliJ and check that it can connect as well

A customer already confirmed that changes working in regards of respecting settings. [[1](https://gitpod.slack.com/archives/C02JV0QE45S/p1656475088222719?thread_ts=1655389525.321889&cid=C02JV0QE45S)]

## Release Notes

```release-note
Respect user settings (proxies, certificates) of JetBrains Gateway while connecting to Gitpod API.
```

## Documentation


## Werft options:
